lib/evmone/baseline.cpp

```diff
@@ -13,6 +13,12 @@ namespace evmone::baseline
 {
 CodeAnalysis analyze(const uint8_t* code, size_t code_size)
 {
+    constexpr auto code_padding = 33;
```

Comment:
Why is it 33? if the last instruction is `PUSH32` then 32 should be enough?

A comment here would be nice.

Reply:
I guess to have yet another 0 for STOP after the pushdata?

Reply:
Ah I see.

Reply:
This is not needed any more.